libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.html

```diff
@@ -323,3 +323,14 @@
     <sky-icon *skyThemeIf="'modern'" icon="trash" size="lg" />
   </button>
 </div>
+
+<div *ngIf="labelText && ngControl?.errors">
```

Comment:
Is the extra `div` needed? Do we need to put `:host { display: block; }` on the `sky-form-errors` component?